examples/NewModuleTemplate/build.psake.ps1

```diff
@@ -273,6 +277,27 @@ Task Test -depends Build {
     }
 }
 
+Task GenerateDocs -depends Build {
+    if ($DefaultLocale -eq $null) {
+        $DefaultLocale = 'en-US'
+    }
+
+    Import-Module "$PublishDir\$ModuleName.psd1" -Global
+    
+    (Get-ChildItem -Path $DocsRootDir -Directory).FullName | ForEach-Object {
+        Update-MarkdownHelp -Path $_ | Out-Null
+    }
+
+    New-MarkdownHelp -Module $ModuleName -Locale $DefaultLocale -OutputFolder "$DocsRootDir\$DefaultLocale" -WithModulePage -ErrorAction SilentlyContinue | Out-Null
```

Comment:
The double quotes on the OutputFolder arg are not necessary. You can remove those.